cluster/examples/kubernetes/ceph/flex/storageclass-test.yaml

```diff
@@ -22,4 +22,4 @@ allowVolumeExpansion: true
 parameters:
   blockPool: replicapool
   clusterNamespace: rook-ceph
-  fstype: xfs
+  fstype: ext4
```

Comment:
Even for flex are we changing the default to ext4?

Reply:
It's a general ceph issue, so the default makes sense for both ceph and flex as I understand.

Reply:
okay, in that case it  makes sense to change in all places :+1: